ios/browser/api/ads/brave_ads.mm

```diff
@@ -363,127 +319,22 @@ - (BOOL)isAdsServiceRunning {
 #pragma mark - Configuration
 
 - (BOOL)isEnabled {
-  return [self.prefs[kRewardsEnabledPrefKey] boolValue];
+  PrefService* pref_service = self.getProfilePrefService;
+  CHECK(pref_service);
```

Comment:
When cleaning up please remove all these checks, `getProfilePrefService` would be removed and you can add the CHECK once on init instead

Reply:
Fixed